osmomath/errors.go

```diff
@@ -0,0 +1,10 @@
+package osmomath
+
+import sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+const codespace = "osmomath"
+
+var (
+	ErrInvalidBaseMustBeUnderTwo = sdkerrors.Register(codespace, 1, "base must be lesser than two")
+	ErrInvalidBaseMustBePositive = sdkerrors.Register(codespace, 2, "base must be positive")
```

Comment:
```suggestion
	ErrInvalidBaseMustBeUnderTwo = sdkerrors.Register(codespace, 2, "base must be lesser than two")
	ErrInvalidBaseMustBePositive = sdkerrors.Register(codespace, 3, "base must be positive")
```

Reply:
Also, is it worth it for these to be sdk errors? I honestly don't think SDK errors are even worth it -- should just be regular `errors.New(...)` I think.